careers/ladder.md

```diff
@@ -28,73 +28,161 @@ IC0 > IC1 > IC2 > IC3 > IC4 > IC5
+
+#### Knowledge Leadership
 - Has some knowledge of CS concepts.
-- Learns to complete well-defined and subdivided tasks.
 - Focuses on learning tools and processes.
 - Pushes code through the entire lifecycle from development to production.
+
+#### Impact
+- Learns to complete well-defined and subdivided tasks.
+
+#### Influence
+
 - Develops individual contributor skills, including source control, editors or test-driven development.
+- Seeks assistance and feedback on a regular basis with teammates and manager
+
+#### Discretion
+- Receives active guidance from peers and manager.
 
 ### IC1 Associate Engineer
 
+#### tl;dr: focuses on being an effective individual contributor
+
+#### Knowledge Leadership
 - Has broad knowledge of CS concepts.
-- Can complete well-defined and subdivided tasks.
 - Focuses on learning tools and processes.
+
+#### Impact
+- Can complete well-defined and subdivided tasks.
+
+#### Influence
 - Routinely pushes code through the entire lifecycle from development to production.
 - Continues to develop individual contributor skills, including source control, editors or test-driven development.
+- Understands team mission, goals and backlog.
+
+#### Discretion
+- Guided by objectives of project or ongoing assignments.
+- Plans own work priorities on a regular basis.
+
 
 ### IC2 Engineer
 
+#### tl;dr: is an effective team member
+
+#### Knowledge Leadership
 - Consistently writes and delivers correct and clean quality code with guidance.
+- Owns a small-to-medium feature from technical design through completion.
+
+#### Impact
+- Makes steady, well-paced progress without the need for constant significant feedback from more senior engineers.
+
+#### Influence
 - Self-sufficient and makes steady progress on tasks.
 - Knows when to ask for help and how to get unblocked.
-- Makes steady, well-paced progress without the need for constant significant feedback from more senior engineers.
-- Owns a small-to-medium feature from technical design through completion.
+- Effectively partners with other engineers.
 - Provides help and support outside of area under their responsibility.
 - Participates in the interviewing process of new team members.
 
+#### Discretion
+- Actively engages in team rituals.
+- Understands business impact of their work.
+
 ### IC3 Senior Engineer
 
-- Owns a service or large technology component.
-- Requires minimal direction or oversight, delivers quickly.
-- Understands elements of the business that their work supports.
+#### tl;dr: can drive change and impact at the team level
+
+#### Knowledge Leadership
 - Understands and makes well-reasoned design decisions and tradeoffs in their area.
 - Works in other areas of the codebase with guidance from owners of that system.
+- Owns a service or large technology component.
+
+#### Impact
+- Delivers significant business value at team level.
+- Helps the team deliver on its overall goals.
+- Can drive specific initiatives.
```

Comment:
I feel like something's missing here about being able to transform ambiguous business requirements into workable projects. I think it's relevant at this level (and not just, e.g., for technical leads or "staff" levels).

Reply:
Should all engineers at the IC3 level be able to fulfill the role of TL if needed? If so, this may be a good opportunity to bake the expectations for a TL into our ladder.

Reply:
I'm not sure if that's the case – our [docs on TL responsibilities] include:

> There is no hard requirement for the level of experience one must have before becoming a tech lead and there’s no need to be a tech lead to reach a certain level.

It seems like, so far anyway, the Tech Lead role is orthogonal to the career ladder.

Reply:
Got it. It sounds like it would be counterproductive to bring the TL role into this discussion.

@joeyAghion sorry for highjacking your thread. I agree that being able to translate business requirements into actionable tasks is an important skill for an IC3 to have so that they can have a big impact their team.

Reply:
I'm happy to call that out more explicitly, it's definitely required to be able to deliver the expected impact.